components/PostImage/PostImage.module.css

```
@@ -1,6 +1,55 @@
 .postImg {
   display: block;
-  max-height: 100%;
+  width: 100%;
+  object-fit: fill;
+  grid-area: center;
+}
+.likeIconContainer {
+  grid-area: center;
+  text-align: center;
+}
+.imgContainer {
+  display: grid;
+  justify-self: center;
+  align-items: center;
+  justify-content: center;
+  grid-template-areas: 'center';
+}
+.likeIcon {
+  color: #fff;
+  transform: scale(3.5);
+  align-self: center;
+  animation: scale 1.5s ease-out;
+}
+@keyframes scale {
+  0% {
+    transform: scale(0);
+  }
+  20% {
+    transform: scale(0);
+  }
+
+  55% {
+    transform: scale(4.5);
+  }
+  59% {
+    transform: scale(5.5);
+  }
+  65% {
+    transform: scale(4.5);
+  }
+  75% {
+    transform: scale(4.5);
+  }
+  85% {
+    transform: scale(4.5);
+  }
+  95% {
+    transform: scale(2.5);
+  }
+  100% {
+    transform: scale(0);
+  }
```

I stole the animation for you from IG :D

```suggestion
   animation: scale 1s ease-in-out;
}
@keyframes scale {
  0%,
  to {
    opacity: 0;
    transform: scale(0);
  }

  15% {
    opacity: 0.9;
    transform: scale(1.2);
  }

  30% {
    transform: scale(0.95);
  }

  45%,
  80% {
    opacity: 0.9;
    transform: scale(1);
  }
}
```